### ISSUE TITLE
Rope winch

### ISSUE THREAD

**Issue by XenonPPG** [studio-minus/ppg-bugs-and-requests#1140]
**Feature description**
Its a new object. Its a winch that extends/shortens the rope. It has a small ball on the end of the rope, so you can connect other things to it

**Feature purpose**
This would be useful for making cranes, grapple hooks, elevators and other things
![Без имени](https://user-images.githubusercontent.com/102830220/163729565-d7ab43df-70c1-466e-bb2e-f5dd2dd954ef.png)

**Comment by DindinYT37** [comment]
Something like this?
![image](https://user-images.githubusercontent.com/64973205/163732017-c3dfa2ba-2cd8-4d07-8c72-7fe34a63f5a9.png)

**Comment by XenonPPG** [comment]
well yeah, but better

**Comment by DindinYT37** [comment]
> well yeah, but better

Yeah, it's indeed better. I tried making a longer version of what I've shown you but attaching things to it is very janky since the different "points" of the rope just glitch through.
![image](https://user-images.githubusercontent.com/64973205/163873971-a8655ca8-9646-4325-bf29-11497e523cad.png)

*I think we need the strength of a wire to not be determined by the weight of where it's originating from, or at least an option for it - same with wire-to-wire connection.*

**Comment by CUBE0R** [comment]
Yes.

**Comment by AmdDoggo** [comment]
best idea
